types/elasticsearch/index.d.ts

```diff
@@ -111,7 +113,7 @@ export interface ConfigOptions {
     keepAlive?: boolean;
     maxSockets?: number;
     suggestCompression?: boolean;
-    connectionClass?: string;
+    connectionClass?: string | typeof HttpConnector;
```

Comment:
Is that right?  Should it be `string | HttpConnector` rather than `string | typeof HttpConnector`?

Reply:
`HttpConnector` is just abtract class for extending or inheriting. So I use `typeof` here. You also see my test in `elasticsearch-test.ts` file for detail. I use `MyHttpConnector` extended from `HttpConnector`.